docker/freebsd-extras.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -x
+set -euo pipefail
+
+main() {
+    local arch="${1}"
+
+    local sqlite_ver=3.33.0,1 \
+          openssl_ver=1.1.1h,1 \
+          target="${arch}-unknown-freebsd12"
+
+    local td
+    td="$(mktemp -d)"
+
+    mkdir "${td}"/{openssl,sqlite}
+
+    pushd "${td}"
+
+    local bsd_arch=
+    case "${arch}" in
+        x86_64)
+            bsd_arch=amd64
+            ;;
+        i686)
+            bsd_arch=i386
+            ;;
+    esac
+
+    # Adding openssl lib
+    curl --retry 3 -sSfL "https://pkg.freebsd.org/FreeBSD:12:${bsd_arch}/quarterly/All/openssl-${openssl_ver}.txz" -O
+    tar -C "${td}/openssl" -xJf openssl-${openssl_ver}.txz /usr/local/lib /usr/local/include/
+
+    # Adding sqlite3
+    curl --retry 3 -sSfL "https://pkg.freebsd.org/FreeBSD:12:${bsd_arch}/quarterly/All/sqlite3-${sqlite_ver}.txz" -O
+    tar -C "${td}/sqlite" -xJf sqlite3-${sqlite_ver}.txz /usr/local/lib
+
+    # Copy the linked library
+    local destdir="/usr/local/${target}"
+    cp "${td}/openssl/usr/local/lib"/lib{crypto,ssl}.a "${destdir}/lib"
+    cp "${td}/openssl/usr/local/lib"/lib{crypto,ssl}.so.11 "${destdir}/lib"
+    cp "${td}/openssl/usr/local/lib"/lib{crypto,ssl}.so "${destdir}/lib"
+    cp -r "${td}/openssl/usr/local/include" "${destdir}"
```

Comment:
Move this to the top.

Reply:
Do you mean `freebsd.sh`? I was debating to have them separate or together since the extras are not 100% required.

Reply:
Do you mean freebsd.sh? I was debating to have them separate or together since the extras are not 100% required.

Reply:
I just mean this line specifically. It's kind of out of place in the middle here.